client/src/js/pages/homePage.js

```diff
@@ -1,12 +1,22 @@
 require('../../styles/pages/home.scss');
 
 const BasePage = require('watch-framework').BasePage;
+const StorageHub = require('watch-framework').StorageHub;
 const logo = require('../../images/logo.png')
 const date = new Date(Date.now()).toLocaleString().split(",")[0];
 const time = new Date(Date.now()).toLocaleString().split(",")[1];
 const compiledTemplate = require("../../templates/homePage.hbs");
 
 class HomePage extends BasePage {
+
+  pageWillLoad() {
+    StorageHub.setData('contacts', [
+      { name: 'Ray', phoneNumber: '0431 111 111' },
+      { name: 'Sinan', phoneNumber: '0431 222 222' },
+      { name: 'Jafari', phoneNumber: '0431 333 333' },
+    ])
+  }
```

Comment:
I've added this concept to get students aware of how to use a StorageHub, and hopefully highlight that it's available from other pages